### DISCUSSION TITLE
angelegte Sachschlagworte 6 - 8 werden in der "Edit Medium" Form nicht angezeigt

### DISCUSSION THREAD

**Discussion by larspetri** [Larian77/Openbiblio#43] [General]
Hi,
ich habe ein neues Medium angelegt und möchte es nun bearbeiten.
In der Form zum Bearbeiten von Medien werden die Schlagworte 6, 7 und 8 nicht angezeigt.

**Comment by Larian77** [comment]
> Hi, ich habe ein neues Medium angelegt und möchte es nun bearbeiten. In der Form zum Bearbeiten von Medien werden die Schlagworte 6, 7 und 8 nicht angezeigt.

Die Standard-Felder, die angezeigt werden müssen derzeit leider noch per Hand in der entsprechenden php-Datei bearbeitet werden, eine Anpassung ist in einer späteren Version vorgesehen. Alternativ bei einem angelegten Medium auf "Marc bearb." gehen und zusätzliche Felder danach definieren und bearbeiten.

**Comment by superudo** [comment]
Hier wäre meine Antwort in Diskussion #31 hilfreich, denke ich.
